src/index.html

```diff
@@ -59,19 +59,50 @@ <h1 class="flex flex-wrap">
             id="alert-facebook"
         ></p>
 
-        <div class="wrapper">
-            <form id="form-profile" accept-charset="UTF-8"></form>
-            <p class="text-center mt-5">
-                <button type="button" id="generate-btn" class="btn btn-primary btn-attestation"><span ><i class="fa fa-file-pdf inline-block mr-1"></i>  Générer mon attestation</span></button>
-            </p>
+    <div class="wrapper">
+        <form id="form-profile" accept-charset="UTF-8"></form>
+        <p class="text-center mt-5">
+            <button type="button" id="generate-btn" class="btn btn-primary btn-attestation">
+              <span>
+                <i class="fa fa-file-pdf inline-block mr-1"></i>
+                Générer mon attestation
+              </span>
+            </button>
+        </p>
 
             <div class="bg-primary  d-none" id="snackbar">
                 L'attestation est téléchargée sur votre appareil.
             </div>
         </div>
 
-        <div id="footnotes">
-            <p id="footnote1">
+        <div class="wrapper">
+          <div id="formgroup-storedata"
+               class="form-check text-center">
+            <input class="form-check-input"
+                   type="checkbox"
+                   name="storedata"
+                   id="field-storedata"
+                   value="storedata">
+            <label class="form-check-label"
+                   for="field-storedata">
+              Stocker mes données sur l'appareil afin de remplir automatiquement mes futures attestations
+            </label>
+          </div>
+        </div>
+
+        <a id="cleardata"
+           class="center delete-data-link"
+           href="javascript:void(0)">Effacer les données du formulaire ?</a>
+        <div class="bg-primary  d-none"
+             id="snackbar-cleardata">
+          Les données du formulaire ont été effacées.
+        </div>
+    </div>
+
+
+    <div class="">
+        <p id="footnotes">
+            <span id="footnote1">
```

Comment:
Je pense qu'il y a quelques soucis dans les balises des footnotes

```suggestion
        <div id="footnotes">
            <p id="footnote1">
```

Reply:
Bien vu ! Une consequence de la copie de l'attestation couvre-feu j'ai l'impression. Je fais un check avant de commit voir s'il n'y aurait pas d'autres balises cassées.